cobalt/build/configs/linux-x64x11/args.gn

```diff
@@ -11,3 +11,7 @@ enable_vulkan = false
 # Overriding flag from //media/media_options.gni.
 # Cobalt doesn't use //third_party/ffmpeg.
 media_use_ffmpeg = false
+
+# Overridden flag from build/config/BUILDCONFIG.gn
+# Cobalt/default toolchain is always clang.
+is_clang = true
```

Comment:
Isn't this true by default?

Reply:
i printed it out on line 162 just outside the declaration box and it's false sometimes: https://paste.googleplex.com/5274026046652416

so good to set it to true by default to avoid unknown behaviour ?

Reply:
hmm, thinking more about this, seems like the default is somehow overriden , either by the toolchain args or something else . 

I did a diff between the 2 print statements : https://diff.googleplex.com/#key=ZV0ixBVi9OiV
there's a visible diff but in both cases is_clang was set to false 15 times  - making me think that the diff is because of the order in which things are run and the values actually might be same .

I'm not sure if we should override this. maybe best to leave as it is so that other chromium toolchain values dont get overwritten unwittingly . wdyt @andrewsavage1 / @kaidokert / @y4vor

Reply:
`is_clang` may be set by secondary toolchains to different values—the thing to check would be if it's true for the default toolchain, which I bet it is

Reply:
it is, even without me declaring the arg the default is clang_x64 toolchain . 

I now realize the original condition was confusing : i read it as this won't be set for linux - wrongly
  is_clang = current_os != "linux" ||
             (current_cpu != "s390x" && current_cpu != "s390" &&
              current_cpu != "ppc64" && current_cpu != "ppc" &&
              current_cpu != "mips" && current_cpu != "mips64" &&
              current_cpu != "riscv64")
              
 I'll remove it